COBOL Programming Course #0 - Getting Started/COBOL Programming Course #0 - Getting Started.md

```diff
@@ -0,0 +1,1085 @@
+---
+geometry:
+- margin=1in
+header-includes:
+- \lstset{basicstyle=\ttfamily,
+    stepnumber=2,
+    numbersep=5pt,
+    backgroundcolor=\color{black!10},
+    showspaces=false,
+    showstringspaces=false,
+    showtabs=false,
+    tabsize=2,
+    captionpos=b,
+    breaklines=true,
+    breakatwhitespace=true,
+    breakautoindent=true,
+    linewidth=\textwidth}
+- \hypersetup{colorlinks=true,
+              linkcolor=blue}
+---
+\newpage
+
+# Getting started {.unnumbered}
```

Comment:
This unnumbered title can be deleted, since it will be a book by itself.

Reply:
Unnumbered title deleted.